blacklist/blacklist.go

```diff
@@ -0,0 +1,94 @@
+package blacklist
+
+import (
+	"github.com/Comcast/codex/db"
+	"github.com/Comcast/webpa-common/logging"
+	"github.com/go-kit/kit/log"
+	"sync"
+	"time"
+)
+
+type List interface {
+	InList(ID string) bool
+}
+
+type memList struct {
+	data     map[string]string
+	dataLock sync.RWMutex
+}
+
+func newEmptyMemList() memList {
```

Comment:
I'm not sure why this is private? `memList` seems like a valid option that a consumer could use or build on if they wanted to; it implements the interface.

Reply:
also I don't think `memList` is exactly accurate for what it accomplishes, maybe `syncList`?